client/src/app/site/motions/components/motion-import-list/motion-import-list.component.ts

```diff
@@ -0,0 +1,265 @@
+import { MatTableDataSource, MatTable, MatSnackBar} from '@angular/material';
+import { BaseViewComponent } from 'app/site/base/base-view';
+import { MotionImportService, NewMotionEntry } from '../../services/motion-import.service';
+import { MotionCsvExportService } from '../../services/motion-csv-export.service';
+
+
+/**
+ * Component for the motion import list view.
+ */
+@Component({
+    selector: 'os-motion-import-list',
+    templateUrl: './motion-import-list.component.html',
+    styleUrls: ['./motion-import-list.component.scss']
+})
+export class MotionImportListComponent extends BaseViewComponent {
+    /**
+     * The data source for a table. Requires to be initialised with a BaseViewModel
+     */
+    public dataSource: MatTableDataSource<NewMotionEntry>;
+
+    public hasFile = false;
+    public selectedEncoding: string;
+    public selectedTextSeparator: string;
+
+    public invalidReasons = {
+        title: 'Motion has no title',
+        text: 'Motion has no text',
+        duplicate: 'A motion with this identifier already exists'
+    };
+
+    /**
+     * indicator on which elements to hide
+     */
+    public hidden = {done: true, errors: false };
+
+    /**
+     * The table itself
+     */
+    @ViewChild(MatTable)
+    protected table: MatTable<NewMotionEntry>;
+
+    /**
+     * Returns the amount of total item successfully parsed
+     */
+    public get totalCount(): number {
+        return (this.importer && this.hasFile) ? this.importer.summary.total : null;
+    }
+
+    public get encodings(): string[] {
+        return this.importer.encodings;
+    }
+
+    public get columnSeparators(): {name: string, value: string}[] {
+        return this.importer.columnSeparators;
+    }
+
+    /**
+     * Returns the amount of import items that will be imported
+     */
+    public get newCount(): number {
+        return (this.importer && this.hasFile) ? this.importer.summary.new : null;
+    }
+
+    public get nonImportableCount(): number {
+        if (this.importer && this.hasFile) {
+            return this.importer.summary.errors + this.importer.summary.duplicates;
+        }
+        return null;
+    }
+
+    public get doneCount(): number {
+        return (this.importer && this.hasFile) ? this.importer.summary.done : null;
+    }
+
+    /**
+     * Constructor for list view bases
+     * @param titleService the title serivce
+     * @param translate the translate service
+     * @param matSnackBar
+     * @param importer: The user csv import service
+     */
+    public constructor( titleService: Title, public translate: TranslateService,
+        matSnackBar: MatSnackBar, private importer: MotionImportService,
+        private motionCSVExport: MotionCsvExportService) {
+        super(titleService, translate, matSnackBar);
+        this.initTable();
+        this.selectedEncoding = this.encodings[0];
+        this.importer.errorEvent.subscribe( message => this.raiseError(message) );
```

Comment:
` this.importer.errorEvent.subscribe(this.raiseError);` should work, becuase `raiseError` is a function that takes one argument.